### PR TITLE
Fix new lints from the latest clippy

### PR DESCRIPTION
**Pull request by Dentosal**
Builds on top of now-closed #695, which is closed in favor of this PR.

### CODE REVIEW

File: forc/src/ops/forc_fmt.rs [hunk 1]
```diff
@@ -222,8 +222,8 @@ name = "Fuel example project"
 
 
 [dependencies]
-core = {git="http://github.com/FuelLabs/sway-lib-core"}
```
Comment (vnepveu) [inline forc/src/ops/forc_fmt.rs:225]: you should add more identation so we actuallly check that the version number gets indented correctly

**Review by vnepveu — COMMENTED** [review]
I'm uncompetent to determine if this should be merged or not

File: forc/src/ops/forc_fmt.rs (hunk 1 above)
Comment (Dentosal) [reply forc/src/ops/forc_fmt.rs:225]: This line verifies that missing whitespace will be added if needed. See the line below this for removing extra whitespace.

Comment (vnepveu) [reply forc/src/ops/forc_fmt.rs:225]: oh ok, smarter than what I had done!

**Review by emilyaherbert — APPROVED** [review]

**Pull request merged**